packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -54,6 +54,7 @@ export default class LocusInfo extends EventsScope {
     this.membership = locus.membership || null;
     this.identities = locus.identities || null;
     this.participants = locus.participants || null;
+    this.deltaParticipants = [];
```

Comment:
Could you add a comment above this line stating what the purpose of this object is? Make sure to include that it mentions that the data is only good for the last locus event.